_theory/maximum-operational-adaptive-capacity.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: under-construction
+title: Maximum operational adaptive capacity
+permalink: /theory/maximum-operational-adaptive-capacity/
+excerpt: TODO
```

Comment:
@Caroline-nc - could you please draft a definition based on the paper?

Reply:
The maximum operational adaptive capacity of a water resource system consists of its theoretical upper limit for adaptation. This theoretical limit is determined based on identifying optimal management policies able to cope with all climate exposures of interest. Beyond this limit, infrastructure upgrades are required to manage future climate changes and reduce a water resource system vulnerability.